platform/windows/os_windows.cpp

```diff
@@ -1356,17 +1356,17 @@ Error OS_Windows::shell_open(String p_uri) {
 }
 
 Error OS_Windows::shell_show_in_file_manager(String p_path, bool p_open_folder) {
-	p_path = p_path.trim_suffix("file://");
+	p_path = p_path.trim_prefix("file://");
```

Comment:
Given that this would never have worked, it might indeed be unnecessary and could be removed I think.

I expect `explorer.exe` would handle a path with `file://` just fine, no?